deploy-manage/distributed-architecture/shard-allocation-relocation-recovery/index-level-shard-allocation.md

```diff
@@ -3,27 +3,147 @@ mapped_urls:
 
-% Scope notes: also reference https://www.elastic.co/guide/en/elasticsearch/reference/current/allocation-total-shards.html and https://www.elastic.co/guide/en/elasticsearch/reference/current/data-tier-shard-filtering.html
+* [Shard allocation filtering](../../../deploy-manage/distributed-architecture/shard-allocation-relocation-recovery/index-level-shard-allocation.md): Controlling which shards are allocated to which nodes.
+* [Delayed allocation](../../../deploy-manage/distributed-architecture/shard-allocation-relocation-recovery/delaying-allocation-when-node-leaves.md): Delaying allocation of unassigned shards caused by a node leaving.
+* [Total shards per node](asciidocalypse://docs/elasticsearch/docs/reference/elasticsearch/index-settings/total-shards-per-node.md): A hard limit on the number of shards from the same index per node.
+* [Data tier allocation](asciidocalypse://docs/elasticsearch/docs/reference/elasticsearch/index-settings/data-tier-allocation-settings.md): Controls the allocation of indices to [data tiers](../../../manage-data/lifecycle/data-tiers.md).
 
-% Use migrated content from existing pages that map to this page:
+## Index-level shard allocation filtering [shard-allocation-filtering]
 
-% - [ ] ./raw-migrated-files/elasticsearch/elasticsearch-reference/index-modules-allocation.md
-%      Notes: conceptual content
-% - [ ] ./raw-migrated-files/elasticsearch/elasticsearch-reference/shard-allocation-filtering.md
-%      Notes: conceptual content
-% - [ ] ./raw-migrated-files/elasticsearch/elasticsearch-reference/recovery-prioritization.md
-%      Notes: conceptual content
+You can use shard allocation filters to control where {{es}} allocates shards of a particular index. These per-index filters are applied in conjunction with [cluster-wide allocation filtering](asciidocalypse://docs/elasticsearch/docs/reference/elasticsearch/configuration-reference/cluster-level-shard-allocation-routing-settings.md#cluster-shard-allocation-filtering) and [allocation awareness](../../../deploy-manage/distributed-architecture/shard-allocation-relocation-recovery/shard-allocation-awareness.md).
 
-⚠️ **This page is a work in progress.** ⚠️
+Shard allocation filters can be based on [custom node attributes](asciidocalypse://docs/elasticsearch/docs/reference/elasticsearch/configuration-reference/node-settings.md#custom-node-attributes) or the built-in `_name`, `_host_ip`, `_publish_ip`, `_ip`, `_host`, `_id`, `_tier` and `_tier_preference` attributes. [Index lifecycle management](../../../manage-data/lifecycle/index-lifecycle-management.md) uses filters based on custom node attributes to determine how to reallocate shards when moving between phases.
```

Comment:
This is now moved from asciidocalypse to elasticsearch repo (https://github.com/elastic/elasticsearch/blob/main/docs/reference/elasticsearch/configuration-reference/node-settings.md).  Ditto for the other places where asciidocalypse is referenced.
```suggestion
Shard allocation filters can be based on [custom node attributes](elasticsearch://reference/elasticsearch/configuration-reference/node-settings.md#custom-node-attributes) or the built-in `_name`, `_host_ip`, `_publish_ip`, `_ip`, `_host`, `_id`, `_tier` and `_tier_preference` attributes. [Index lifecycle management](../../../manage-data/lifecycle/index-lifecycle-management.md) uses filters based on custom node attributes to determine how to reallocate shards when moving between phases.
```